gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -1140,6 +1144,7 @@ gst_tensor_converter_parse_caps (GstTensorConverter * self,
       self->in_info.video = info;
       break;
     }
+#ifndef NO_AUDIO
     case _NNS_AUDIO:
     {
       GstAudioInfo info;
```

Comment:
If this is the only code that uses gst-audio*.so, it'd be better use runtime-if rather than #ifdef.
And use #ifdef only inside this case statement, inducing negotiation-failures with AUDIO inputs.